src/EnergyPlus/MixedAir.cc

```diff
@@ -3648,6 +3617,7 @@ void VentilationMechanicalProps::CalcMechVentController(
     // new local variables for DCV
     // Zone OA flow rate based on each calculation method [m3/s]
     std::array<Real64, static_cast<int>(DataSizing::OAFlowCalcMethod::Num)> ZoneOACalc{0.0};
+    Real64 CO2PeopleGeneration = 0; // CO2 generation from people at design level
```

Comment:
Keeping this here at the top, will move while addressing #9969

Reply:
Understood